client/apps/edit/components/edit_container.jsx

```diff
@@ -0,0 +1,105 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import * as Actions from 'client/actions/editActions'
+
+import { EditContent } from './content/index.jsx'
+import { EditDisplay } from './display/index.jsx'
+import { EditHeader } from './header/index.jsx'
+import EditAdmin from './admin/index'
+
+class EditContainer extends Component {
+  static propTypes = {
+    article: PropTypes.object,
+    actions: PropTypes.object,
+    channel: PropTypes.object,
+    edit: PropTypes.object,
+    user: PropTypes.object
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      lastUpdated: null
+    }
+
+    props.article.sections.on(
+      'change add remove reset',
+      () => this.maybeSaveArticle()
+    )
+  }
+
+  onChange = (key, value) => {
```

Comment:
the onChange actions can still be moved into redux, but for now they are at least consolidated to one place.